backend/routes/public/items.php

```diff
@@ -0,0 +1,23 @@
+<?php
+$req_method = "GET";
+require '../../includes/header.php';
+
+$sql = "SELECT id,name,image_url FROM items WHERE status=1";
+
+$result = $db->query($sql);
+if ($result->num_rows == 0) {
+    $items = 0;
+} else {
+    $items = array();
+    while ($row = $result->fetch_assoc()) {
+        $items[$row['id']]['name'] = $row['name'];
```

Comment:
Ja bym zrobił po prostu Array a id byłoby atrybutem obiektu w arrayu
Bo trochę łatwiej na froncie po tym iterować

Ale to taki szczegół i w sumie może być jak jest